packages/create-jest/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "create-jest",
+  "description": "Create a new Jest project",
+  "version": "29.6.4",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jestjs/jest.git",
+    "directory": "packages/create-jest"
+  },
+  "license": "MIT",
+  "bin": "./bin/create-jest.js",
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./build/index.d.ts",
+      "default": "./build/index.js"
+    },
+    "./package.json": "./package.json",
+    "./bin/create-jest": "./bin/create-jest.js"
```

Comment:
It does not export a thing, so can be skipped. Or I missed something?

```suggestion
```

Reply:
I'm not sure also, borrowed this from `jest-cli`

Reply:
Ok, I checked, we have specific assertion for that [here](https://github.com/dj-stormtrooper/jest/blob/feat/create-jest-package/scripts/buildUtils.mjs#L63)